bin/runtime/src/lib.rs

```diff
@@ -758,6 +759,93 @@ impl pallet_identity::Config for Runtime {
     type RegistrarOrigin = EnsureRoot<AccountId>;
     type WeightInfo = pallet_identity::weights::SubstrateWeight<Self>;
 }
+parameter_types! {
+    // Key size = 32, value size = 8
+    pub const ProxyDepositBase: Balance = 40 * LEGACY_DEPOSIT_PER_BYTE;
+    // One storage item (32) plus `ProxyType` (1) encode len.
+    pub const ProxyDepositFactor: Balance = 33 * LEGACY_DEPOSIT_PER_BYTE;
+    // Key size = 32, value size 8
+    pub const AnnouncementDepositBase: Balance =  40 * LEGACY_DEPOSIT_PER_BYTE;
+    // AccountId, Hash and BlockNumber sum up to 68
+    pub const AnnouncementDepositFactor: Balance =  68 * LEGACY_DEPOSIT_PER_BYTE;
+}
+#[derive(
+    Copy,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Encode,
+    Decode,
+    RuntimeDebug,
+    MaxEncodedLen,
+    scale_info::TypeInfo,
+)]
+pub enum ProxyType {
+    Any = 0,
+    NonTransfer = 1,
+    Staking = 2,
+}
+impl Default for ProxyType {
+    fn default() -> Self {
+        Self::Any
+    }
+}
+impl InstanceFilter<RuntimeCall> for ProxyType {
+    fn filter(&self, c: &RuntimeCall) -> bool {
+        match self {
+            ProxyType::Any => true,
+            ProxyType::NonTransfer => matches!(
+                c,
+                RuntimeCall::Staking(..)
+                    | RuntimeCall::Session(..)
+                    | RuntimeCall::Treasury(..)
+                    | RuntimeCall::Vesting(pallet_vesting::Call::vest { .. })
+                    | RuntimeCall::Vesting(pallet_vesting::Call::vest_other { .. })
+                    | RuntimeCall::Vesting(pallet_vesting::Call::merge_schedules { .. })
+                    | RuntimeCall::Utility(..)
+                    | RuntimeCall::Multisig(..)
+                    | RuntimeCall::NominationPools(..)
+                    | RuntimeCall::Identity(..)
+                    | RuntimeCall::Proxy(..)
```

Comment:
I read the code of the pallet a bit and I would propose removing `Proxy` from the list. Weird shit can happen if you allow proxies here. And we don't want to get bitten in the ass for no benefit.

Reply:
ok, removed